projects/configs/tag-operations/base.yaml

```
@@ -0,0 +1,424 @@
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# Default configuration for Standard MR Image
+
+- name: "Private Creator Data Element"
+  group: 0x000d
+  element: 0x0010
+  op: "keep"
+- name: "UCLH Project Name"
+  group: 0x000d
+  element: 0x1001
+  op: "keep"
+- name: "Specific Character Set"
+  group: 0x0008
+  element: 0x0005
+  op: "keep"
+- name: "Image Type"
+  group: 0x0008
+  element: 0x0008
+  op: "keep"
+- name: "Instance Creator UID"
+  group: 0x0008
+  element: 0x0014
+  op: "replace_UID"
+- name: "SOP Class UID"
+  group: 0x0008
+  element: 0x0016
+  op: "keep"
+- name: "SOP Instance UID"
+  group: 0x0008
+  element: 0x0018
+  op: "replace_UID"
+- name: "Accession Number"
+  group: 0x0008
+  element: 0x0050
+  op: "replace"
+- name: "Study Date"
+  group: 0x0008
+  element: 0x0020
+  op: "replace"
+- name: "General Study"
+  group: 0x0008
+  element: 0x0030
+  op: "replace"
+- name: "Modality"
+  group: 0x0008
+  element: 0x0060
+  op: "keep"
+- name: "Modalities In Study"
+  group: 0x0008
+  element: 0x0061
+  op: "keep"
+- name: "Manufacturer"
+  group: 0x0008
+  element: 0x0070
+  op: "keep"
+- name: "Referring Physician's Name"
+  group: 0x0008
+  element: 0x0090
+  op: "replace"
+- name: "Study Description"
+  group: 0x0008
+  element: 0x1030
+  op: "keep"
+- name: "Series Description"
+  group: 0x0008
+  element: 0x103e
+  op: "keep"
+- name: "Manufacturers Model Name"
+  group: 0x0008
+  element: 0x1090
+  op: "keep"
+- name: "Referenced Series Sequence"
+  group: 0x0008
+  element: 0x1155
+  op: "replace_UID"
+- name: "Referenced Frame Number"
+  group: 0x0008
+  element: 0x1160
+  op: "keep"
+- name: "Pixel Presentation"
+  group: 0x0008
+  element: 0x9205
+  op: "keep"
+- name: "Volumetric Properties"
+  group: 0x0008
+  element: 0x9206
+  op: "keep"
+- name: "Volume Based Calculation Technique Attribute"
+  group: 0x0008
+  element: 0x9207
+  op: "keep"
+- name: "Complex Image Component"
+  group: 0x0008
+  element: 0x9208
+  op: "keep"
+- name: "Acquisition Contrast"
+  group: 0x0008
+  element: 0x9209
+  op: "keep"
+- name: "Patients Name"
+  group: 0x0010
+  element: 0x0010
+  op: "replace"
+- name: "Patient ID"
+  group: 0x0010
+  element: 0x0020
+  op: "secure-hash"
+- name: "Patient's Birth Date"
+  group: 0x0010
+  element: 0x0030
+  op: "replace"
+- name: "Patient's Sex"
+  group: 0x0010
+  element: 0x0040
+  op: "keep"
+- name: "Scanning Sequence"
+  group: 0x0018
+  element: 0x0020
+  op: "keep"
+- name: "Sequence Variant"
+  group: 0x0018
+  element: 0x0021
+  op: "keep"
+- name: "MR Image"
+  group: 0x0018
+  element: 0x0022
+  op: "replace"
```

I think this should be called this
```suggestion
- name: "Scan Options Attribute"
  group: 0x0018
  element: 0x0022
  op: "replace"
```